source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py

```diff
@@ -453,6 +452,9 @@ def __init__(
         api_composition.decorate_iqs_client(self, root_configuration)
         self.jupyter_tools = ext_point.IQSJupyterTools(self)
 
+    def __get__(self, instance, owner):
+        return owner.connect()
```

Comment:
I do not really understand what this is supposed to do. I can't really map its behaviour onto the specification of `__get__`:

- https://docs.python.org/3/reference/datamodel.html#object.__get__ 
- https://python-reference.readthedocs.io/en/latest/docs/dunderdsc/get.html

Have you tested it carefully?

Do we have a reason to think that the _class_ `owner` would have a `connect()` function (not any instance method)? Also, I think `__get__` should be defined in a property descriptor class associated with the public property (it may still delegate to a private backing field); it is weird to add it to the actual type of object you want to return. This may cause unexpected behaviour if someone stores the `iqs` object in an object field in the future.

Less important: why do we not call the file `iqs` for simplicity?

Reply:
I think I've got the wrong idea with this method... it seems that it is not necessary. In the code I used 
```python
connector.iqs_client.server_management.get_server_info()
```
while the Login button always sets the `iqs_client` property upon clicking, it shall affect most use cases.